docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -54,7 +54,7 @@ quarkus.http.cors.access-control-allow-credentials=true <7>
 ----
 
 <1> Enables the CORS filter.
-<2> Specifies allowed origins, including a regular expression.
+<2> Specifies allowed origins, including a regular expression. Leaving this value empty blocks all origins.
```

Comment:
I don't mind having the information here but is it also in the config doc for the property?